### DISCUSSION TITLE
Bump updated_at on parent resource

### DISCUSSION THREAD

**Discussion by janwillemvd** [directus/directus#14333] [Feature Requests]
Currently, when you have a relational trigger which nullifies the parent when a referenced child is deleted from a parent record, the following happens at db level:

- the child record is deleted 
- the referenced child column on the parent's record is set to null
- the updated_at on the parent table is untouched

The first two steps are expected, but I was hoping the third step would also occur. However, it doesn't.

Is this intentional or shall I start working on a PR which does update all parent's columns which have the following setting:

![Screenshot 2022-07-08 at 16 08 56](https://user-images.githubusercontent.com/10515908/178008500-cf1a07d2-7c4f-4572-818f-4bc56ef6a84c.png)

**Comment by rijkvanzanten** [comment]
This is kinda intentional, as those relational triggers are handled on the database directly, rather than on the Directus side of things. Therefore, Directus isn't aware that change is made, and the `updated_on` trigger isn't fired 🤔 

The only way to get around that would be to re-implement all the relational trigger logic in Directus itself, which is a non-trivial amount of work

**Reply by janwillemvd** [reply]
I see, thanks for the prompt reply!

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
